kikit/fab_ui.py

```diff
@@ -98,6 +98,25 @@ def oshpark(**kwargs):
     app = fakeKiCADGui()
     return execute(oshpark.exportOSHPark, kwargs)
 
+@click.command()
+@fabCommand
+@click.option("--assembly/--no-assembly", help="Generate files for SMT assembly (schematics is required)")
```

Comment:
Since the goal is to export for PnP machine, I kinda miss the point of allowing not to export the assembly data.

Reply:
Removed assembly switch and force to provide schematic.